Stripe/STPTheme.m

```diff
@@ -201,8 +217,8 @@ - (id)copyWithZone:(__unused NSZone *)zone {
     copyTheme.secondaryForegroundColor = self.secondaryForegroundColor;
     copyTheme.accentColor = self.accentColor;
     copyTheme.errorColor = self.errorColor;
-    copyTheme.font = self.font;
-    copyTheme.emphasisFont = self.emphasisFont;
+    copyTheme->_font = _font;
+    copyTheme->_emphasisFont = _emphasisFont;
```

Comment:
I was gonna ask to use dot-access to set these but I think that would break the dynamic getter?  

I guess we should `copy` these since that's what the property attributes say.

Reply:
ah good idea

Reply:
and yes re the dynamic getter :)